dts/bindings/pinctrl/microchip,xec-pinctrl-conf.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020 Linaro Limited
+# Copyright (c) 2021 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
```

Comment:
where is this file used now?

Reply:
This file is no longer used, will be removed.